flink-table/flink-table-planner/src/test/java/org/apache/flink/table/api/CompiledPlanITCase.java

```diff
@@ -417,4 +426,11 @@ private File createSourceSinkTables() throws IOException {
         createTestCsvSourceTable("src", DATA, COLUMNS_DEFINITION);
         return createTestCsvSinkTable("sink", COLUMNS_DEFINITION);
     }
+
+    private String getPreparedToCompareCompiledPlan(final String planAsString) throws IOException {
+        return TableTestUtil.getPrettyJson(
```

Comment:
in case of pretty json it is much faster to find the diff than in case of everything in one line...